weave-js/src/components/Panel2/PanelGroup.tsx

```diff
@@ -658,11 +658,13 @@ export const PanelGroupItem: React.FC<{
 
           // This updates the grid config with the new name, since we use names as ids
           // if we had unique ids, we wouldnt have to do this
-          const gridConfigIndex = config.gridConfig.panels.findIndex(
-            p => p.id === name
-          );
-          if (gridConfigIndex !== -1) {
-            draft.gridConfig.panels[gridConfigIndex].id = newName;
+          if (config.gridConfig != null) {
```

Comment:
this is a smell - why is this not initialized?

Reply:
not sure if something changed but the reason varnames stopped being editable, before shawns ui change, was that vertical groups stopped having gridconfigs